From 046a2c0102ef683c1d8e2a6719037d2b9d16739d Mon Sep 17 00:00:00 2001
From: Daniel Borkmann <dborkman@redhat.com>
Date: Tue, 22 Jan 2013 10:24:32 +0100
Subject: [PATCH 27/53] sctp_darn: remove never read peer_prim_len variable
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Fixes:

  sctp_darn.c: In function ‘peer_primary_func’:
  sctp_darn.c:1963:6: warning: variable ‘peer_prim_len’ set but not used [-Wunused-but-set-variable]

Signed-off-by: Daniel Borkmann <dborkman@redhat.com>
---
 src/apps/sctp_darn.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/src/apps/sctp_darn.c b/src/apps/sctp_darn.c
index 9843a8a..551e49d 100644
--- a/src/apps/sctp_darn.c
+++ b/src/apps/sctp_darn.c
@@ -1960,14 +1960,13 @@ peer_primary_func(char *argv0, int sk, char *cp, int set)
 	struct sctp_setpeerprim setpeerprim;
 	struct sockaddr_in *in_addr;
 	struct sockaddr_in6 *in6_addr;
-	int peer_prim_len, ret;
+	int ret;
 	char *p = cp;
 
 	if (!set) {
 		goto err;
 	}
 
-	peer_prim_len = sizeof(struct sctp_setpeerprim);
 	/* Set the buffer for address parsing.  */
 	while ('\n' != *p)
 		p++;
-- 
1.7.11.7

